public/js/base/ui.js

```diff
@@ -62,8 +61,8 @@ define(["jquery", "text!html/ui-fragments.html"], function ($, _fragments) {
   UI.pagination = function (page, total, limit, callback) {
     var $pagination = $(".pagination"),
       $ul = $pagination.find("ul"),
-      $_li = $("<li></li>"),
-      $li,
+      $li = $("<li></li>"),
```

Comment:
if we're renaming here anyway, we can probably pick better names. What do $li and $liClone mean in terms of their role on the page?